### DISCUSSION TITLE
After upgrading the percussioncms-tools, the DevOptions file is backed up and replaced, is this intentional?

### DISCUSSION THREAD

**Discussion by kuldeepmalik** [percussion/percussioncms#578] [Q&A] [answered]
1. Upgrade devtool.
2. Navigate to Install_Dir\devtools.
3. Observe that DevOptions.cmd.bak is showing with the customized values saved before upgrade and a new DevOptions batch file is showing with the default values.
![image](https://user-images.githubusercontent.com/77011198/157620446-2de2dae1-f617-4a73-80c0-ea5bf67779f4.png)

**Comment by natechadwick** [comment, marked as answer]
This is by design.  In the event we have to change the developer options file, we back it up and replace it on upgrade.  The developer can then re-apply their options from the backup file after upgrade.
